boards/arm/nrf52_adafruit_feather/doc/nrf52_adafruit_feather.rst

```diff
@@ -0,0 +1,178 @@
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth            |
++-----------+------------+----------------------+
+| RTT       | on-chip    | console              |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+See `Adafruit Feather nRF52 Bluefruit Learning Guide`_ and `Nordic Semiconductor Infocenter`_
+for a complete list of nRF52 Adafruit Feather board hardware features.
+
+Connections and IOs
+===================
+
+The `Adafruit Feather nRF52 Bluefruit Learning Guide`_ has detailed
+information about the board including `pinouts`_ and the `schematic`_.
+
+LED
+---
+
+* LED0 (red) = P0.17
+* LED1 (blue) = P0.19
+
+Push buttons
+------------
+
+* DFU = SW0 = P0.20
+* RESET = SW1 = P0.21/reset
+
+Programming and Debugging
+*************************
+
+Applications for the ``nrf52_adafruit_feather`` board configuration can be
+built and flashed in the usual way (see :ref:`build_an_application`
+and :ref:`application_run` for more details); however, the standard
+debugging targets are not currently available.
+
+Flashing
+========
+
+Follow the instructions in the :ref:`nordic_segger` page to install
+and configure all the necessary software. Further information can be
+found in :ref:`nordic_segger_flashing`. Then build and flash
+applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Here is an example for the :ref:`hello_world` application.
+
+
+#. Build the Zephyr kernel and the :ref:`hello_world` sample application:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/hello_world
+      :board: nrf52_adafruit_feather
+      :goals: build
+      :compact:
+
+#. Connect the Adafruit nRF52 Feather to your host computer using USB
+
+#. Run your favorite terminal program to listen for output. 
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the nRF52 Adafruit Feather
+board can be found. For example, under Linux, :code:`/dev/ttyUSB0`.
+
+#. Flash the image:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/hello_world
+      :board: nrf52_adafruit_feather
+      :goals: flash
+      :compact:
+
+   You should see "Hello World! arm" in your terminal.
+
+
+Debugging
+=========
+
+Refer to the :ref:`nordic_segger` page to learn about debugging Nordic boards with a
+Segger IC.
+
+
+Testing the LEDs and buttons on the nRF52 Adafruit Feather
+**********************************************************
+
+There are several samples that allow you to test that the buttons (switches) and LEDs on
+the board are working properly with Zephyr:
+
+.. code-block:: console
```

Comment:
Rather than a list like this, it would be better to reference the sample documents, so replace the code block and contents with this list of doc links:

    - :ref:`blinky-sample`
    - :ref:`button-sample`
    - :ref:`fade-led-sample`
    - :ref:`blink-led-sample`
    - :ref:`96b_carbon_multi_thread_blinky`